retrieval/discovery/ec2.go

```diff
@@ -29,10 +31,15 @@ import (
 
 const (
 	ec2Label           = model.MetaLabelPrefix + "ec2_"
+	ec2LabelAZ         = ec2Label + "az"
```

Comment:
Let's keep this in line with the response we receive, and call this `availability_zone`